### ISSUE TITLE
chimera.vsearch_error

### ISSUE THREAD

**Issue by Pratima-c** [mothur/mothur#684]
Hi, 
I tried to run chimera.vsearch and I got an error
 saying error in fasta file and accession denied. I looked at the fasta file and it looks good. 

Could you please help me to figure out the issue?
Your help will be appreciated. 
Thanks.

This is what I did
mothur > chimera.vsearch(fasta=current, count=current, dereplicate=t)
Using 191030_pratima.files1.trim.contigs.pcr.good.unique.good.filter.precluster.count_table as input file for the count parameter.
Using 191030_pratima.files1.trim.contigs.pcr.good.unique.good.filter.precluster.fasta as input file for the fasta parameter.

Using 8 processors.
Checking sequences from 191030_pratima.files1.trim.contigs.pcr.good.unique.good.filter.precluster.fasta ...
Error in reading your fastafile, at position -1. Blank name.

Removing group: Feed1 because all sequences have been removed.

Removing group: Feed2 because all sequences have been removed.

Removing group: Feed3 because all sequences have been removed.

Removing group: Feed4 because all sequences have been removed.

and so on. It removed sequences from all samples....

**Comment by mothur-westcott** [comment]
Is the 191030_pratima.files1.trim.contigs.pcr.good.unique.good.filter.precluster.fasta file empty? Can you run the following?

mothur > summary.seqs(fasta=191030_pratima.files1.trim.contigs.pcr.good.unique.good.filter.precluster.fasta, count=191030_pratima.files1.trim.contigs.pcr.good.unique.good.filter.precluster.count_table)

**Comment by Pratima-c** [comment]
No, it was not empty. I ran summary.seqs. The error was in vsearch. I
redownloaded the new vsearch and reran the command and it worked. I don't
know why I got the error saying error in fasta file. But now problem has
been solved.
Thank you .

........
Best Regards
Pratima Chapagain
*Graduate** Teaching Assistant*
*Department of Biology, Office SCI 2093*
*Middle Tennessee State University*
*Murfreesboro, TN 37132*
*Email:chapagainpratima0@gmail.com /pc3d@mtmail.mtsu.edu*
